### PR TITLE
[ONNX] GlobalAveragePool

### PR DESCRIPTION
**Pull request by cynthia-liu**

### CODE REVIEW

**Comment by cynthia-liu** [comment]
GlobalAveragePool is needed in ONNX model: densenet121

**Comment by cynthia-liu** [comment]
ZOO-PR-Validation - Build # 1774 - Successful:

Check console output at http://10.239.47.211:18888/job/ZOO-PR-Validation/1774/ to view the results.

The pull request: https://github.com/intel-analytics/analytics-zoo/pull/1085 Successful

File: pyzoo/zoo/pipeline/api/onnx/mapper/globalaveragepool.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class GlobalAveragePoolMapper(OperatorMapper):
+    def __init__(self, node, _params, _all_tensors):
+        super(GlobalAveragePoolMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        x = self.model_inputs[0].zvalue
+        y = zautograd.expand_dims(zlayers.GlobalAveragePooling2D()(x), 2)
+        return zautograd.expand_dims(y, 3)
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/globalaveragepool.py]: add comments for why expand dim here?

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 2]
```diff
@@ -1325,3 +1325,17 @@ def test_shape(self):
...
+
+    def test_globalaveragepool(self):
+        node = onnx.helper.make_node(
+            'GlobalAveragePool',
+            inputs=['x'],
+            outputs=['y'],
+        )
+        x = np.random.randn(2, 3, 7, 5).astype(np.float32)
+        spatial_shape = np.ndim(x) - 2
+        y = np.average(x, axis=tuple(range(spatial_shape, spatial_shape + 2)))
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: add a pytorch unitest as well

**Comment by cynthia-liu** [comment]
Jenkins passed
Check console output at http://10.239.47.211:18888/job/ZOO-PR-Validation/2037/ to view the results.

**Pull request merged**